sdk/communication/communication-phone-numbers/test/internal/customPipelinePolicies.spec.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I see you testing the policies individually here, but I'm wondering if we should test them in the client itself.